README.md

```diff
@@ -29,11 +29,11 @@ You'll need [Bazel >= 0.27][bazel-getting-started] installed.
 In a fresh directory, run:
 
 ```console
-$ curl https://haskell.build/start | sh
+$ sh <(curl https://haskell.build/start) --use-bindists
```

Comment:
The common case should remain as simple and as memorable as possible. The common is bindists, so that should be the default, if no flag is passed. That would allow keeping the instructions on this line identical to what they were.

We can mention in text displayed to the user that there is also a `--use-nixpkgs` flag that they might want to use, and how to use it.